x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -217,11 +217,12 @@ const ActionsComponent: React.FC<ActionProps> = ({
   );
 
   const onExpandEvent = useCallback(() => {
-    if (isTourAnchor) {
+    const isStep2Active = activeStep === 2 && isTourShown(SecurityStepId.alertsCases);
```

Comment:
![1](https://user-images.githubusercontent.com/6935300/198685903-0ad375b8-266a-4921-aa8b-5a7528f52e51.gif)
To prevent bugs like this, only allow `incrementStep` from this action if the alerts tour step is currently 2

Reply:
use the enum here instead of 2 and maybe `isExpandEventStepActive` ?